src/workingtitle-vcockpits-instruments-navsystems/html_ui/WorkingTitle/Pages/VCockpit/Instruments/NavSystems/Shared/CommonPFD_MFD.js

```diff
@@ -1100,7 +1100,7 @@ class PFD_WindData extends NavSystemElement {
         this.mode = WTDataStore.get("WindData.Mode", 0);
     }
     init(root) {
-        this.svg = root;
```

Comment:
I assume this is probably the same in all planes, but might want to check others just in case.

Reply:
Ah, good catch.   I'll check on that.

Reply:
Looks like it is the same in all planes, but I still decided that that was a poor way of doing this.   I set it back to what it was and just subclassed PFD_WindData in the G3X to add the different logic for detecting the SVG element.